.github/workflows/E2E integration.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         if: runner.os == 'Windows'
         
       - name: Run eslint
-        run: npx eslint 
+        run: npx eslint@7.32.0 
```

Comment:
this should use the version in package.json... i'm not sure why we're installing the latest packages above and not using the ones defined in the repo